SConstruct

```diff
@@ -190,6 +190,7 @@ opts.Add(BoolVariable('builtin_squish', "Use the built-in squish library", True)
 opts.Add(BoolVariable('builtin_thekla_atlas', "Use the built-in thekla_altas library", True))
 opts.Add(BoolVariable('builtin_zlib', "Use the built-in zlib library", True))
 opts.Add(BoolVariable('builtin_zstd', "Use the built-in Zstd library", True))
+opts.Add(BoolVariable('builtin_certs', "Bundle default SSL certificates to be used if you don't specify an override in the project settings", True))
```

Comment:
Those options are listed alphabetically ;)

Reply:
Butting in to say that I don't think that option fits in this section anyways -- this is for third-party libraries, and certs aren't a library in the same sense.